### PR TITLE
New language: Latvian.

### PR DESCRIPTION
**Pull request by davislasis**
New language: Latvian.

### CODE REVIEW

File: Language/lv/Seed.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+// Seed language strings
+return [
+    // Seed command
+    'nameFile'   => 'Nosauciet SEEDER failu.',
+    'writeError' => 'Mēģinot izveidot failu {0}, radās kļūda. Pārbaudiet, vai direktorija ir rakstāma.',
+];
```
Comment (paulbalandan) [inline Language/lv/Seed.php:17]: Add final new line.

**Review by paulbalandan — CHANGES_REQUESTED** [review]

File: Language/lv/Seed.php (hunk 1 above)
Comment (davislasis) [reply Language/lv/Seed.php]: Done

**Review by paulbalandan — APPROVED** [review]

**Comment by davislasis** [comment]
@paulbalandan why JA language keys are merged few hours ago [https://github.com/codeigniter4/translations/pull/124](url)
and my PR now fails because of that?
Even if i would need to add those keys in LV, Travis don't fail with missing keys
[https://travis-ci.org/github/codeigniter4/translations/jobs/745687868](url)
[https://travis-ci.org/github/codeigniter4/translations/jobs/745687869](url)

Either way there is something wrong, could you please guide the best approach through this?

**Comment by paulbalandan** [comment]
Ok. So that's why Travis didn't run on JA. Either way it's not your fault as while I'm waiting for this test to finish I merged that. I am currently coming up with a test case that will just test the locale of the changed files. I  am just looking for the best way to retrieve the list of changed files in a PR. Also, the added JA keys should not be a worry as before this gets a release alongside the 4.1 release these keys should have been added to all locales.

**Pull request merged**